### PR TITLE
2sv

### PR DESCRIPTION
**Pull request by fillup**

### CODE REVIEW

File: terraform/060-simplesamlphp/main.tf [hunk 1]
```diff
@@ -78,6 +79,7 @@ data "template_file" "task_def" {
     mysql_user                  = "${var.mysql_user}"
     recaptcha_key               = "${var.recaptcha_key}"
     recaptcha_secret            = "${var.recaptcha_secret}"
+    remember_me_secret          = "${var.remember_me_secret}"
```
Comment (forevermatt) [inline terraform/060-simplesamlphp/main.tf]: Do you also need to add `remember_me_secret` to the README.md file?

Comment (fillup) [reply terraform/060-simplesamlphp/main.tf]: oh yeah, guess so

**Comment by fillup** [comment]
Yes, another good catch, I'll update that to be safe.

**Comment by fillup** [comment]
Actually looks like it is fine as is. in `000-core` we're using an `app_name` like `idp-appsdev` since it isn't one of the specific app service environments.

**Pull request merged**